web/docs/operators/deduplicate.md

```diff
@@ -17,35 +17,59 @@ deduplicate [<extractor>...]
 
 ## Description
 
-The `deduplicate` operator deduplicates values over a stream of events, based
+The `deduplicate` operator removes duplicates from a stream of events, based
 on the value of one or more fields.
 
+You have three independent configuration options to customize the operator's
+behavior:
+
+1. **Limit**: the multiplicity of the events until they are supressed as
+   duplicates. A limit of 1 is equivalent to emission of unique events. A limit
+   of *N* means that events with a unique key (defined by the fields) get
+   emitted at most *N* times. For example, `AAABAABBAC` with a limit of 2 yields
+   `AABBC`.
+2. **Distance**: The number of events in sequence since the last occurrence of
+   a unique event. For example, deduplicating a stream `AABABACBABB` with
+   distance 3 yields `ABC`.
```

Comment:
I'd use a distance of two for the example, as anything that's three or higher yields the same result.